exchanges/okx/okx_test.go

```diff
@@ -2264,7 +2264,7 @@ func TestGetOrderHistory(t *testing.T) {
 	_, err := ok.GetOrderHistory(contextGenerate(), &getOrdersRequest)
 	if err == nil {
 		t.Errorf("Okx GetOrderHistory() Expected: %v. received nil", err)
-	} else if err != nil && !errors.Is(err, errMissingAtLeast1CurrencyPair) {
+	} else if !errors.Is(err, errMissingAtLeast1CurrencyPair) {
 		t.Errorf("Okx GetOrderHistory() Expected: %v, but found %v", errMissingAtLeast1CurrencyPair, err)
 	}
```

Comment:
This makes the 2265 test redundant, since nil is not errMissingAtLeast1CurrencyPair.

Suggest the whole thing becomes:
```suggestion
	assert.ErrorIs(t, err, errMissingAtLeast1CurrencyPair, "GetOrderHistory should error when not passed any pair")
```

DriveBy: That's a horrible error message name. Sounds like errPairRequired to me, and we probably should have a common.Err for that, I'd imagine. Or currency.Err